### PR TITLE
Add script for moving Jox holdings to Jo

### PR DESCRIPTION
**Pull request by mxtthias**

### CODE REVIEW

File: whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+String NEW_SIGEL = 'https://libris.kb.se/library/Jo'
+
+PrintWriter scheduledForMove = getReportWriter("scheduled-for-move")
+PrintWriter notMoved = getReportWriter("refused-to-move")
+
+selectBySqlWhere("""
+        collection = 'hold' AND
+        data#>>'{@graph,1,heldBy,@id}' = '${CURRENT_SIGEL}'
+    """, silent: false) { hold ->
+    def (record, thing, work) = hold.graph
```
Comment (niklasl) [inline whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy]: No work expected for hold (Items). Changing to `(record, thing)` would be good.

File: whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    selectBySqlWhere("""
+        collection = 'hold' AND
+        data#>>'{@graph,1,itemOf,@id}' = '${thing["itemOf"][ID]}' AND
+        data#>>'{@graph,1,heldBy,@id}' = '$NEW_SIGEL'
+    """, silent: true) { other_hold ->
+        wouldCreateDupe = true
+    }
+
+    if (wouldCreateDupe) {
```
Comment (niklasl) [inline whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy]: Do we really want to keep these Jox-holdings?

Comment (mxtthias) [reply whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy]: Yup. See comment in ticket.

File: whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy (hunk 1 above)
Comment (mxtthias) [reply whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy]: I fix!

File: whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy (hunk 2 above)
Comment (niklasl) [reply whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy]: All good then! :white_check_mark:

**Pull request merged**